.pre-commit-config.yaml

```diff
@@ -0,0 +1,15 @@
+repos:
+  - repo: https://github.com/google/yapf
```

Comment:
Are there other formatter tools that we should be using?

Reply:
Addressed in top-level PR comment.